build jdk on mac and ubuntu hit ##[error]The process '/bin/bash' failed with exit code 1

build jdk on mac and ubuntu hit ##[error]The process '/bin/bash' failed with exit code 1:

./mk-cacerts.sh: line 79: '/home/runner/work/build-jdk/build-jdk/jdk/boot'/bin/keytool: No such file or directory
5110
##[error]The process '/bin/bash' failed with exit code 1

https://github.com/AdoptOpenJDK/build-jdk/runs/1114107682?check_suite_focus=true

--- a/src/builder.ts
+++ b/src/builder.ts
@@ -85,7 +85,7 @@ export async function buildJDK(
   }
 
   await exec.exec(`bash ./makejdk-any-platform.sh \
-  -J '${jdkBootDir}' \
+  -J "${jdkBootDir}" \
   --configure-args "${configureArgs}" \
   -d artifacts \
   --target-file-name ${fullFileName} \

